components/style/gecko/media_queries.rs

```diff
@@ -35,8 +37,15 @@ pub struct Device {
     pub pres_context: RawGeckoPresContextOwned,
     default_values: Arc<ComputedValues>,
     viewport_override: Option<ViewportConstraints>,
+    /// The font size of the root element
+    /// This is set when computing the style of the root
+    /// element, and used for rem units in other elements
```

Comment:
Please mention why it's fine to use it like this, that is:

When computing the style of the root element, there can't be any other style being computed at the same time, given we need the style of the parent to compute everything else.